### PR TITLE
Update comments and logging

### PR DESCRIPTION
**Pull request by Anandkumar26**

### CODE REVIEW

**Comment by reachjainrahul** [comment]
nephe-test-e2e-kind

File: pkg/controllers/networkpolicy/sync.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func (a *appliedToSecurityGroup) sync(syncContent *securitygroup.Synchronization
 		np := i.(*networkPolicy)
 		if !np.rulesReady {
 			if !np.computeRules(r) {
-				log.V(1).Info("np not ready", "Name", np.Name, "Namespace", np.Namespace)
+				log.V(1).Info("Np is not ready", "Name", np.Name, "Namespace", np.Namespace)
```
Comment (reachjainrahul) [inline pkg/controllers/networkpolicy/sync.go:103]: Change this to NetworkPolicy

Comment (Anandkumar26) [reply pkg/controllers/networkpolicy/sync.go]: Done

**Pull request merged**